jsx/Filter.js

```diff
@@ -39,7 +39,8 @@ class Filter extends Component {
   onFieldUpdate(name, value, id, type) {
     const searchParams = new URLSearchParams(location.search);
     const filter = JSON.parse(JSON.stringify(this.props.filter));
-    const exactMatch = type === 'textbox' ? false : true;
+    let exactMatch = type === 'textbox' ? false : true;
```

Comment:
i do think this functionality is needed though. but it should be done like this:

```suggestion
    let exactMatch = (type === 'textbox' || type === 'date') ? false : true;
```

Reply:
and line 980 of `Form.js` changed to `this.props.onUserInput(this.props.name, e.target.value, e.target.id, 'date');`

Reply:
Thanks @zaliqarosli, I didn't think to make it one line like that. I made it a little shorter by doing: `const exactMatch = (!(type === 'textbox' || type === 'date'));`